itm/itm_adm_scenario_runner.py

```diff
@@ -192,8 +192,15 @@ def get_random_supply(self, state: State):
         supplies = [new_supply.type for new_supply in state.supplies if new_supply.quantity > 0]
         return random.choice(supplies)
 
-    def get_random_character_id(self):
-        return random.choice(self.adm_knowledge.all_character_ids)
+    def get_random_character_id(self, action_type):
+        if action_type in [ActionTypeEnum.MOVE_TO_EVAC]:
+            characters : List[Character] = [character for character in self.adm_knowledge.characters]
```

Comment:
I didn't see this portion mentioned in the slack discussion but may have missed it, sorry if that is the case. But my initial thought is if we have to `MOVE_TO` the character to perform other actions like treatments on them, it seems more intuitive to me that we would have to `MOVE_TO` that character before `MOVE_TO_EVAC`.

Reply:
Yes, with the creation of "locations" and `MOVE_TO`, ideally `MOVE_TO_EVAC` could be renamed because it's more about choosing who should be evacuated (when evacuation is ready) instead of actually/physically *moving* them into a helicopter, e.g.   There's no code to remove the character from the scene once you've "moved" him to evac.

But [somewhere along the way](https://inthemoment-group.slack.com/archives/C05AQMCA5PB/p1716480617894449?thread_ts=1715363899.873879&cid=C05AQMCA5PB), ADEPT did specify that you can select an unseen character to "move" to evac-- although they were considering it in terms of "intending" to evac an unseen patient, but as written, that's exactly what `MOVE_TO_EVAC` does.